src/stories/containers/Finances/components/FinacesTable/FinancesTable.tsx

```diff
@@ -29,6 +29,10 @@ const FinancesTable: React.FC<Props> = ({ className, breakdownTable, metrics, pe
   const showQuarterly = !isMobile && period === 'Quarterly';
   const showMonthly = desk1440 && period === 'Monthly';
   const arrayMetrics = new Array<number>(iteration).fill(0);
+  const showFooter = true;
+  // Show color for others depending if odd or even
+  const isPar = orderData[`${tables[tables.length - 1]}`]?.length % 2 === 0;
```

Comment:
could we rename this var to keep it in English :)

Reply:
you are right, done